docs/source/user_guide/automl.ipynb

```diff
@@ -651,11 +651,131 @@
+    "                                  engine = threaded_cf_engine)\n",
+    "automl_cf_threaded.search(show_iteration_plot = False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: the cell demonstrating process-level parallelism is commented out due to incompatibility with our ReadTheDocs build.  It can be run successfully locally.\n",
+    "\n",
+    "```python\n",
+    "from concurrent.futures import ProcessPoolExecutor\n",
+    "\n",
+    "# Repeat the process but using process-level parallelism\n",
+    "process_cf_engine = CFEngine(CFClient(ProcessPoolExecutor()))\n",
+    "\n",
+    "automl_cf_process = AutoMLSearch(X_train=X, y_train=y,\n",
+    "                                 problem_type=\"binary\",\n",
+    "                                 engine = process_cf_engine)\n",
+    "automl_cf_process.search(show_iteration_plot = False)\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Parallelism with Dask\n",
+    "\n",
+    "Thread or process level parallelism can be explicitly evoked for the `DaskEngine` (as well as the `CFEngine`).  The `processes` can be set to `True` and the number of processes set using `n_workers`.  If `processes` is set to `False`, then the resulting parallelism will be threaded and `n_workers` will represent the threads used.  Examples of both will follow."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dask.distributed import Client, LocalCluster\n",
+    "\n",
+    "from evalml.automl.engine import DaskEngine\n",
+    "\n",
+    "dask_engine_p2 = DaskEngine(Client(LocalCluster(processes=True, n_workers = 2)))\n",
+    "automl_dask_p2 = AutoMLSearch(X_train=X, y_train=y,\n",
+    "                              problem_type=\"binary\",\n",
+    "                              allowed_model_families=[ModelFamily.LINEAR_MODEL],\n",
+    "                              engine = dask_engine_p2)\n",
+    "automl_dask_p2.search(show_iteration_plot = False)\n",
+    "del(dask_engine_p2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dask_engine_t4 = DaskEngine(Client(LocalCluster(processes=False, n_workers = 4)))\n",
+    "\n",
+    "automl_dask_t4 = AutoMLSearch(X_train=X, y_train=y,\n",
+    "                              problem_type=\"binary\",\n",
+    "                              allowed_model_families=[ModelFamily.LINEAR_MODEL],\n",
+    "                              engine = dask_engine_t4)\n",
+    "automl_dask_t4.search(show_iteration_plot = False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As we can see, a significant performance gain can result from simply using something other than the default `SequentialEngine`."
```

Comment:
Can you be more specific about what sort of performance gain comes from this?

Reply:
Hmmm, I'd like to, but I'm a little hesitant to specifically quantify something that can vary a little bit with different runs.